src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/ConfigurationClientExtensions.cs

```diff
@@ -166,6 +169,9 @@ await foreach (ConfigurationSetting setting in client.GetConfigurationSettingsAs
                         Label = options.Label.NormalizeNull(),
                         Current = null
                     });
+                    string key = kvp.Key.Replace(FeatureManagementConstants.FeatureFlagMarker, "");
```

Comment:
We should use [Substring](https://learn.microsoft.com/en-us/dotnet/api/system.string.substring?view=netstandard-2.0#system-string-substring(system-int32)) instead. 
Substring will be faster because we are not searching for anything in the string. Moreover, `Replace` will replace all occurrences of FeatureFlagMarker from the key.